api/HTMLInputElement.json

```diff
@@ -1543,7 +1550,7 @@
               "version_added": "50"
             },
             "firefox_android": {
-              "version_added": "50"
+              "version_added": false
```

Comment:
This looked a bit suspect, but confirmed with http://mdn-bcd-collector.appspot.com/tests/api/HTMLInputElement that `webkitEntries` isn't enabled. It is behind a "dom.webkitBlink.filesystem.enabled" pref if you'd like to add it, but I won't block this PR on it.